universal-login-react/src/app/valueRounder.ts

```diff
@@ -0,0 +1,17 @@
+import {IS_STRING_A_NUMBER} from '@unilogin/commons';
+
+export class ValueRounder {
+  private static isNumber(value?: string): boolean {
+    return (typeof (value) === 'string' && !!value.match(IS_STRING_A_NUMBER));
```

Comment:
can use implementation of santitizers: https://github.com/EthWorks/restless/tree/master/restless

Reply:
```ts
  const result = asNumber(value)
  if (Result.isOk(result)) {
    return true
 } else {
    return false; 
 }
```